src/api/app/views/webui/status_messages/new.html.haml

```diff
@@ -0,0 +1,24 @@
+- @pagetitle = 'New Status Message'
```

Comment:
To have a consistent naming, let's use `Create Status Message`

Reply:
done!